src/libs/actions/Report.js

```diff
@@ -343,18 +348,18 @@ function openReport(reportID, participantList = [], newReportObject = {}) {
         onyxData.optimisticData[0].onyxMethod = CONST.ONYX.METHOD.SET;
 
         // Also create a report action so that the page isn't endlessly loading
```

Comment:
Is this comment still relevant? If it is, I am confused by it.

Reply:
Hmm I would expect we create the action everywhere now so maybe the comment becomes redundant with the context of this refactor

Reply:
So I should remove this line?